tests/filter_performance_png_test.php

```diff
@@ -34,7 +34,7 @@ class filter_performance_png_test extends advanced_testcase {
     protected $xml;
     protected $imagepng;
 
-    protected function setUp() {
+    protected function setUp(): void {
```

Comment:
Is this void clause interfering with the Moodle versions that don't need it?

Reply:
It doesn't affect the other supportet versions